cluster/helm/splice-validator/templates/validator.yaml

```diff
@@ -129,9 +129,17 @@ spec:
             canton.validator-apps.validator_backend.scan-client.seed-urls = [ ${_scan.admin-api.address} ]
             {{- end }}
         {{ end }}
-        {{ if not .Values.useSequencerConnectionsFromScan }}
+        {{ if not .Values.useSequencerConnectionsFromScan -}}
         - name: ADDITIONAL_CONFIG_STATIC_SEQUENCER_URL
           value: canton.validator-apps.validator_backend.domains.global.url = {{ .Values.decentralizedSynchronizerUrl | quote }}
+        {{ else if .Values.sequencerClient -}}
+            {{- if .Values.sequencerClient.url}}
```

Comment:
I think we also need a type on the sequencer-client, which defaults to `from-scan`, and then have the other possible values `trusted-url` and `trusted-svs`.